src/i18.rs

```diff
@@ -0,0 +1,20 @@
+
+use fluent_resmgr::resource_manager::ResourceManager;
+use sys_locale::get_locale;
+use unic_langid::LanguageIdentifier;
+
+pub fn translate(change: &str) -> String{
+    let locale = get_locale().unwrap_or_else(|| String::from("en-US"));
+
+    let langid: LanguageIdentifier = locale.parse().expect("wrong language"); 
+    let locales = vec![langid.into()];
+    let resources = vec!["message.ftl".into()];
+    let mgr = ResourceManager::new("./translations/{locale}/{res_id}".into()); 
+    let bundle = mgr.get_bundle(locales, resources);
```

Comment:
Instead of having to create a new bundle every time we call `translate`, we should only create the bundle once. 

We should use a `OnceCell` to lazily initialize the a `ResoruceBundle` the first time it is needed, and then re-use that instance afterwards.

Reply:
I may need to research this area again. It seemed like setting a global variable so that it could only be called once did not work.

Reply:
I think you would have something like

```rust

static resource_manager:  OnceCell<ResourceManager> = OnceCell::new();
static bundle: OnceCell<FluentBundle<&'static FluentResource> = OnceCell::new();


# and then in this function you would have something like:

let mgr = resource_manager.get_or_init(|| {
   // we still need to fix this path...
   ResourceManager::new("./translations/{locale}/{res_id}".into()); 
});

let bundle = bundle.get_or_init(|| {
  let locale = get_locale().unwrap_or_else(|| String::from("en-US"));
   let langid: LanguageIdentifier = locale.parse().expect("wrong language"); 
    let locales = vec![langid.into()];
    let resources = vec!["message.ftl".into()];
   mgr.get_bundle(locales, resources)
});